### ISSUE TITLE
The different steps to implement and complete the project

### ISSUE THREAD

**Issue by harshithsaiv** [harshithsaiv/NormCompressAI#1]
Follow these steps to complete the implementation of the project 

- [x]  Define the autoencoder architecture
- [x]  Load and preprocess the CIFAR-10 dataset
- [x]  Initialize the model , loss function and optimizer
- [x]  Train the model

**Comment by harshithsaiv** [comment]
Completed the implementation
